<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaListarMotorista</class>
 <widget class="QDialog" name="TelaListarMotorista">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonVoltar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblMotorista">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>50</y>
     <width>651</width>
     <height>331</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Alterar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>640</y>
     <width>126</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Alterar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CNH">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>520</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CNH - CARTEIRA NACIONAL DE HABILITAÇÃO</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>590</y>
     <width>236</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_NumeroResgistro">
      <property name="text">
       <string>Número de Registro</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_NumeroRegistro">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>590</y>
     <width>172</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_DataValidade">
      <property name="text">
       <string>Data de Validade</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEditData"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>440</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Nome do Motorista</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_NomeMotorista"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>440</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>RG </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_RG_Motorista">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>440</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_CPF">
      <property name="text">
       <string>CPF</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtCPF">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>400</y>
     <width>204</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="labelNomeMot">
      <property name="text">
       <string>Nome Cliente</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtNomeC">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>186</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Telefone </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Telefone_Motorista"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>165</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Email_Motorista"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>590</y>
     <width>188</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Categoria</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditCategoria"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>200</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID Motorista</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditID"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
